### DISCUSSION TITLE
Semantic release does not updates package.json version, but writes that it does on build

### DISCUSSION THREAD

**Discussion by alex-clear** [semantic-release/semantic-release#2509] [Configuration]
### Current behavior

I faced with issue with semantic release that it was canceled to update our package.json  version update after I updated the version to latest 19.0.2.
My build actions are:
<img width="940" alt="Screen Shot 2022-07-18 at 9 15 38" src="https://user-images.githubusercontent.com/97834715/179454278-01370427-d26f-4ce7-972a-eb5e459928a3.png">
I know that the latest version is 19.0.3, but I got noticed about the issue only now and as i see it started after last upgrade to 19.0.2.

### Expected behavior

The version in. package.json is updated on butid

### `semantic-release` version

19.0.2

### CI environment

Github

### `semantic-release` configuration

Here is my .releaserc.js: 
<img width="272" alt="Screen Shot 2022-07-18 at 9 10 43" src="https://user-images.githubusercontent.com/97834715/179454035-adf9c8ed-e0b6-41c6-9059-dd95bb8a654f.png">

### CI logs

 It writes in logs that version was updated by in package.json I see an old version.
<img width="1321" alt="Screen Shot 2022-07-18 at 9 09 28" src="https://user-images.githubusercontent.com/97834715/179454023-38a547bb-b6fd-4602-aa75-afa02c8276c3.png">

**Comment by travi** [comment]
https://semantic-release.gitbook.io/semantic-release/support/faq#why-is-the-package.jsons-version-not-updated-in-my-repository

**Reply by azabraao** [reply]
<img width="724" alt="image" src="https://user-images.githubusercontent.com/17601387/181646778-cc97da2c-0638-4534-a7e5-e63b8beaf255.png">

@travi does this basically mean that we need to choose between updating in the package.json OR in the npm registry?

**Reply by travi** [reply]
The npm plugin is responsible for publishing. The git plugin is only for committing changes that result from the release process back to the repo. If you want both, you need both plugins. 

However, committing the `package.json` changes to the repo is unnecessary for publishing to npm, so we normally recommend against the complexity unless you _really_ want it.
